public/app/features/explore/Logs/LogsTable.tsx

```diff
@@ -48,6 +48,9 @@ export function LogsTable(props: Props) {
 
   const prepareTableFrame = useCallback(
     (frame: DataFrame): DataFrame => {
+      if (!frame || !frame.length) {
+        return frame;
+      }
```

Comment:
Given the type `frame` should always be trueish, so I think that can be reduced to 

```suggestion
      if (!frame.length) {
        return frame;
      }
```